### PR TITLE
fix: adding optional workload value to pod-failover

### PR DESCRIPTION
**Pull request by connorbradshaw10**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
adding ability to specify workload type to pod failover tests

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

**Review by abhisheksinghbaghel — APPROVED** [review]

File: test/e2e/manifest/pod-failover.json [hunk 1]
```diff
@@ -52,4 +52,4 @@
             }
         }
     }
-}
\ No newline at end of file
+}
```
Comment (mukhoakash) [inline test/e2e/manifest/pod-failover.json:55]: This might create some linting issue, you can add back the empty line EOF.

**Review by landreasyan — APPROVED** [review]

**Pull request merged**